shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Address.hs

```diff
@@ -338,6 +340,20 @@ getByron =
     Left e -> fail (show e)
     Right r -> pure $ AddrBootstrap $ BootstrapAddress r
 
+-- | The size of the extra attributes in a bootstrp (ie Byron) address. Used
+-- to help enforce that people do not post huge ones on the chain.
+bootstrapAddressAttrsSize :: BootstrapAddress crypto -> Int
+bootstrapAddressAttrsSize (BootstrapAddress addr) =
+  -- I'm sorry this code is formatted so weridly below.
+  -- It is to apease the capricious god Ormolu. A sacrifice is demanded!
+  maybe
+    0
+    (BS.length . Byron.getHDAddressPayload)
+    (Byron.aaVKDerivationPath (Byron.attrData attrs))
+    + Byron.unknownAttributesLength attrs
+  where
+    attrs = Byron.addrAttributes addr
```

Comment:
Ok, on the 10th iteration I think I have satisfied the capricious one. Beauty has been sacrificed as demanded.

Reply:
:hocho: